pkg/block/blocktest/adapter.go

```diff
@@ -145,6 +145,7 @@ func testAdapterWalker(t *testing.T, adapter block.Adapter, storageNamespace str
 					prefix = testPrefix
 				}
 
+				require.Lenf(t, results, filesAndFolders*filesAndFolders+1, "expected %d results, got %d", filesAndFolders*filesAndFolders+1, len(results))
```

Comment:
[Current Go style](https://go.dev/wiki/TestComments#got-before-want) is "got ... want ..." not "expected ... got ...".  Both words and order.

Reply:
Also I would expect an assertions library to print a good message for this case, why do we need to repeat the values?

Reply:
will populate expected results and compare with the actual results - will make it more informative in case of mismatch results.